x-pack/platform/plugins/shared/alerting/server/application/rule/methods/bulk_delete/bulk_delete_rules.ts

```diff
@@ -195,6 +197,21 @@ const bulkDeleteWithOCC = async (
     await untrackRuleAlerts(context, id, attributes as RawRule);
   }
 
+  const eventLogClient = await context.getEventLogClient();
+  await pMap(
+    rulesToDelete.map((rule) => rule.id),
+    (ruleId) =>
+      disableGaps({
```

Comment:
we should have a bulk disable gaps function to reduce the number of requests made - so we probably need `processAllGaps` to accept an array of rule IDs to make it bulk as well